examples/miscellaneous-examples.md

```diff
@@ -43,23 +43,26 @@ message.channel.send('What tag would you like to see? This will await will be ca
 Discord quietly changed the Create Guild API endpoint, small bots \(10 guilds or fewer\) are able to create guilds programmatically now. This example will have your bot create a new guild and create a role with the administrator permission, and the single line of code at the bottom will apply it to you when you execute it when you join the guild.
 
 ```javascript
+const { Permissions } = require('discord.js');
+
 /* ES6 Promises */
-client.guilds.create('Example Guild', 'london').then(guild => {
+client.guilds.create('Example Guild').then(guild => {
   guild.channels.cache.get(guild.id).createInvite()
     .then(invite => client.users.cache.get('<USERID>').send(invite.url));
-  guild.roles.create({ name:'Example Role', permissions:['ADMINISTRATOR'] })
+  guild.roles.create({ name: 'Example Role', permissions: Permissions.FLAGS.'ADMINISTRATOR' })
```

Comment:
This should not be a string, and will throw. I'm sure you just missed it ;)

```suggestion
  guild.roles.create({ name: 'Example Role', permissions: Permissions.FLAGS.ADMINISTRATOR })
```

Reply:
I certainly did ;)